### PR TITLE
Update albatross_parameters.m

### PR DESCRIPTION
**Pull request by Jack78000**
Use function to calculate drag of aircraft, for both clean and with external payload configurations

### CODE REVIEW

File: albatross_parameters.m [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
 
 S   = 68;        %m2
 AR  = 18;            
 b   = sqrt(S*AR); %m
 c   = S/b;
 
-
+% Aerodynamic Parameters
 e   = 0.85;
-cdo = 0.03;
```
Comment (NigelLowe) [inline albatross_parameters.m]: Ben's code needs the cdo variable. We'll update that file as well when this change is made.

Comment (NigelLowe) [inline albatross_parameters.m]: Ben's code needs the cdo variable. We'll update that file as well when this change is made.

Comment (Jack78000) [reply albatross_parameters.m]: oh ok sorry didnt realise. Whats the difference between cd0 and cdo?

Comment (NigelLowe) [reply albatross_parameters.m]: cdo decreases when payload dropped, but since its iterating in Bens code, in the next loop, it needs to start with cd0

**Pull request closed without merging**